postgres/manage_users.py

```diff
@@ -4,21 +4,27 @@
 from botocore.exceptions import ClientError
 
 
-def drop_user(cursor, username):
+def create_or_update_db_user(conn, cursor, username, password, permissions):
     cursor.execute(f"SELECT * FROM pg_catalog.pg_user WHERE usename = '{username}'")
 
     if cursor.fetchone() is not None:
-        cursor.execute(f"DROP OWNED BY {username}")
-        cursor.execute(f"DROP USER {username}")
+      update_db_user_password(conn, cursor, username, password)  
+    else:
+      create_db_user(conn, cursor, username, password, permissions) 
 
 
-def create_db_user(conn, cursor, username, password, permissions):
-    drop_user(cursor, username)
+def update_db_user_password(conn, cursor, username, password):
+    cursor.execute(f"ALTER USER {username} WITH ENCRYPTED PASSWORD '%s'" % password)
+    conn.commit()
+    
 
+def create_db_user(conn, cursor, username, password, permissions):
+    MASTER_USERNAME = "postgres"
+    APP_USERNAME = "application_user"
```

Comment:
Doesn't this function also get called with readonly user?

```suggestion
    APP_USERNAME = username
```